faunadb-java-dsl/src/main/java/com/faunadb/client/query/Language.java

```diff
@@ -1215,6 +1316,15 @@ public static Expr Select(Expr path, Expr from, Expr defaultValue) {
     return Fn.apply("select", path, "from", from, "default", defaultValue);
   }
 
+  /**
+   * Creates a new Select expression.
+   *
+   * @see <a href="https://fauna.com/documentation/queries#misc_functions">FaunaDB Miscellaneous Functions</a>
+   */
+  public static Expr SelectAll(Expr path, Expr from) {
```

Comment:
Why not a optional parameter to `Select`?

Reply:
I discussed with @freels and he thinks that it was a mistake adding that argument to select function so we decided to add that sugar and maybe in the future turn it in a function itself.